packages/paste-core/components/page-header/stories/index.stories.tsx

```diff
@@ -0,0 +1,226 @@
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable import/no-extraneous-dependencies */
+import { Anchor } from "@twilio-paste/anchor";
+import { Avatar } from "@twilio-paste/avatar";
+import { Badge } from "@twilio-paste/badge";
+import { Box } from "@twilio-paste/box";
+import { Breadcrumb, BreadcrumbItem } from "@twilio-paste/breadcrumb";
+import { Button } from "@twilio-paste/button";
+import { ButtonGroup } from "@twilio-paste/button-group";
+import { CustomizationProvider } from "@twilio-paste/customization";
+import { DetailText } from "@twilio-paste/detail-text";
+import { InPageNavigation, InPageNavigationItem } from "@twilio-paste/in-page-navigation";
+import { Paragraph } from "@twilio-paste/paragraph";
+import {
+  ProgressStepComplete,
+  ProgressStepCurrent,
+  ProgressStepIncomplete,
+  ProgressStepSeparator,
+  ProgressSteps,
+} from "@twilio-paste/progress-steps";
+import { useTheme } from "@twilio-paste/theme";
+import * as React from "react";
+
+import {
+  PageHeader,
+  PageHeaderActions,
+  PageHeaderContext,
+  PageHeaderDetailText,
+  PageHeaderDetails,
+  PageHeaderHeading,
+  PageHeaderInPageNavigation,
+  PageHeaderMeta,
+  PageHeaderParagraph,
+  PageHeaderPrefix,
+} from "../src";
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: "Components/Page Header",
+  component: PageHeader,
+};
+
+export const Default = (): React.ReactElement => {
+  return (
+    <PageHeader size="default">
+      <PageHeaderContext>
+        <Breadcrumb>
+          <BreadcrumbItem href="#">Breadcrumb</BreadcrumbItem>
+          <BreadcrumbItem href="#">Breadcrumb</BreadcrumbItem>
+        </Breadcrumb>
+      </PageHeaderContext>
+      <PageHeaderDetails>
+        <PageHeaderDetailText>
+          <DetailText marginTop="space0">Detail text</DetailText>
+        </PageHeaderDetailText>
+        <PageHeaderPrefix>
+          <Avatar size="sizeIcon90" name="Avatar Name" />
+        </PageHeaderPrefix>
+        <PageHeaderHeading>Page title</PageHeaderHeading>
+        <PageHeaderActions>
+          <ButtonGroup>
+            <Button variant="secondary">Action</Button>
+            <Button variant="secondary">Action</Button>
+          </ButtonGroup>
+        </PageHeaderActions>
+        <PageHeaderMeta>
+          <Badge variant="success" as="span">
+            Badge
+          </Badge>
+          <DetailText marginTop="space0">Meta</DetailText>
+          <DetailText marginTop="space0">
+            <Anchor href="#">Meta</Anchor>
+          </DetailText>
```

Comment:
To get around this (customers remembering to wrap Anchor in Detail Text), @SiTaggart suggested setting a size, weight and color on PageHeaderMeta: https://www.figma.com/file/BiaSrZ2lqhWSd0rnrDxon2/Page-Header-exploration?type=design&node-id=161-38&mode=design&t=vCf9oSePWplObfXx-4

Reply:
(This should also fix the Meta in the compact variant story)